content/en/logs/archives/reloading/s3.md

```diff
@@ -0,0 +1,151 @@
+
+{{< img src="logs/archives/log_historical_views.png" alt="Reload from Archives" responsive="true" style="width:75%;">}}
+
+Complete the reload form and select "Reload from Archives" to initiate the reload job. The reloaded results will go to their own special index called a "Historical View" and will have all existing facets applied. If you wish to reload from an archive that no longer exists, [follow these instructions to create that archive again][1].
+
+{{< img src="logs/archives/log_reload_from_archives.png" alt="Reload from Archives" responsive="true" style="width:75%;">}}
+
+Once you have started a reload job, the job will appear as "pending" in [the Historical Views tab][4]. When it completes it will appear as "active" and will link you to the log explorer where you can find the reloaded log events.
+
+How long a reload job takes to completed depends on the volume of data that exists in the archive for the time-range queried on, and on the count of log events that match the reload query. Reloads over large archives or large time periods can take hours to complete.
+
+## Private Beta Limitations
+
+While Archive Reloading is in private beta, the following limitations apply:
+
+1. You can only reload up to 300 Million log events with one reload job.
+2. You can only reload archived content that is older than 24 hours. 
+
+## Reload Query Rules
+
+Reload query filters can include any of the following arguments:
+
+* `service`
+* `source`
+* `status`
+* `host`
+* any attribute beginning with `@`
+* raw-text searches (which are applied on the `message` attribute)
+
+The following rules currently apply to what reload queries are supported:
+
+1. Reloading by tags is not yet supported.
+2. Use of wildcards is supported.
+3. Special characters (`+ - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /`) [must be properly escaped][7].
+4. Querying by `service` is required.
+
+### Reload Query Examples
+
+For example, the following reload query will reload all log events from the "foo" and "bar" services whose `http.url_details.path` attribute includes the string "my/endpoint" and whose message includes the string "GET":
+
+`service:(foo OR bar) @http.url_details.path:*my\/endpoint* GET`
+
+The following reload queries are not currently supported:
+
+* `@http.url_details.path:*my\/endpoint* GET` <br>(missing `service` argument)
+* `service:(foo OR bar) env:prod @http.url_details.path:*my\/endpoint* GET` <br>(uses a tag, `env`, or else does not properly escape `:` in the raw text search for `env:prod`, <br>should be `env\:prod` or `"env:prod"`)
+* `service:(foo OR bar) GET http://myapp:1234/users` <br>(does not properly escape the `:` and `/` characters in the raw text search for `GET http://myapp:1234/users`, <br>should be `GET http\:\/\/myapp\:1234\/users` or `"GET http://myapp:1234/users"`)
+
+## Historical View Names
```

Comment:
This should actually be in the main part.

instead of calling it "starting an archive reloading job", we should define a historical view and how to set it up (a bit like saved views for instance).